### PR TITLE
Added 2 videos, fixed FA

### PR DESCRIPTION
**Pull request by Jakub-KK**
Adding 2 videos, both are longer ones with lots of stuff so the link is to the timestamp with the route.
Fixed FA ([source](https://www.ukclimbing.com/news/2013/02/stoking_the_fire_-_another_9b_from_sharma-67799))

### CODE REVIEW

File: data/lead.json [hunk 1]
```diff
@@ -222,7 +223,7 @@
 	{
 		"name": "Stoking the Fire",
 		"grade": "9b",
-		"fa": "Adam Ondra",
+		"fa": "Chris Sharma",
 		"repeat": ["Stefano Ghisolfi", "Sachi Amma"],
```
Comment (9cpluss) [inline data/lead.json]: Thanks for spotting, could you please add Adam Ondra to the repeat row? He actually did the [2nd ascent](https://www.ukclimbing.com/news/2016/02/stoking_the_fire_9b_by_ondra-70276).

**Review by 9cpluss — CHANGES_REQUESTED** [review]
Thanks for adding the videos and spotting the error.

File: data/lead.json (hunk 1 above)
Comment (Jakub-KK) [reply data/lead.json]: Done! How did I missed that? :)

**Comment by 9cpluss** [comment]
Thank you :)

**Pull request merged**